docs/generated/packages/eslint-plugin/documents/dependency-checks.md

```diff
@@ -26,7 +30,7 @@ You can use the `dependency-checks` rule by adding it to your ESLint rules confi
 }
 ```
 
-Linting `JSON` files is not enabled by default, so you will also need to add `package.json` to the `lintFilePatterns`:
+Additionally, you need to add the `package.json` to the `lintFilePatterns`:
```

Comment:
Optional, but I'd slightly rewrite to:

"Additionally, you need to adjust your `lintFilePatterns` to include the project's `package.json` file:"